docs/index.md

```diff
@@ -1,5 +1,39 @@
-# Jetson Car
+# Autonomous car powered by Nvidia Jetson Nano
+
+Nvidia Jetson Nano controlled car with camera
 
 This little 1:12 model car is controlled by a Nvidia Jetson nano to drive autonomounsly with object detection.
 
-![Jetson Car](car_base.jpg)
+This project was planned at AISVN at the end of 2019 to help with machine learning, object detection, inference, computer vision (OpenCV) and artificial intelligence. It uses the 128 CUDA cores of the Nvidia Jetson Nano.
+
+## Hardware
+
+Base is this car model unit from [hshop.vn](https://hshop.vn/products/khung-xe-robot-kim-loai-racing-car-1) for 790.000 VND.
```

Comment:
### Hardware source

Good idea to include the link. Don't forget to update it and post a date of purchease!